{"filter":false,"title":"simple.py","tooltip":"/my_package/src/simple.py","undoManager":{"mark":-1,"position":-1,"stack":[]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":4,"column":17},"end":{"row":4,"column":23},"isBackwards":true},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1547096730000,"hash":"a75f00013f16b8561faa4d9c3097e8cd461cb3eb"}